docs/esps/sendgrid.rst

```diff
@@ -407,13 +453,45 @@ in either Anymail settings or esp_extra as described above.)
 Status tracking webhooks
 ------------------------
 
-If you are using Anymail's normalized :ref:`status tracking <event-tracking>`, enter
-the url in your `SendGrid mail settings`_, under "Event Notification":
+Anymail's normalized :ref:`status tracking <event-tracking>` works
+with SendGrid's webhooks.
 
-   :samp:`https://{random}:{random}@{yoursite.example.com}/anymail/sendgrid/tracking/`
+SendGrid optionally provides webhook signature verification. You have three
+choices for securing the status tracking webhook:
 
-     * *random:random* is an :setting:`ANYMAIL_WEBHOOK_SECRET` shared secret
-     * *yoursite.example.com* is your Django site
+* Use SendGrid's signature verification: follow their
+  `Event Webhook Security Features`_ documentation and set
+  :setting:`SENDGRID_TRACKING_WEBHOOK_VERIFICATION_KEY <ANYMAIL_SENDGRID_TRACKING_WEBHOOK_VERIFICATION_KEY>`
+  (requires the :pypi:`cryptography` package---see :ref:`sendgrid-installation`)
```

Comment:
```suggestion
  (requires the :pypi:`cryptography` package --- see :ref:`sendgrid-installation`)
```

Reply:
(I was always told not to put spaces around em dashes, though I gather both styles are common. Anymail's docs don't really have a style guide, but none of the other em dashes have spaces.)